src/main/java/weizberg/citibike/aws/CitibikeRequestHandler.java

```diff
@@ -20,7 +21,7 @@ public CitibikeResponse handleRequest(APIGatewayProxyRequestEvent event, Context
         CitibikeRequest request = gson.fromJson(body, CitibikeRequest.class);
 
         MergeStationData mergeStationData = new MergeStationData();
```

Comment:
This needs to be a member variable.
Otherwise you create StationCache for each request because it's inside MergeStationData